function-server/src/main/java/io/dispatchframework/javabaseimage/Entrypoint.java

```diff
@@ -4,56 +4,32 @@
 ///////////////////////////////////////////////////////////////////////
 package io.dispatchframework.javabaseimage;
 
+import org.springframework.context.annotation.Configuration;
+
 /**
  * Main entry point for starting the function handling server
  */
 public class Entrypoint {
 
-    /**
-     * @param args
-     * @throws Exception
-     */
     public static void main(String[] args) throws Exception {
 
-        boolean springInClasspath = true;
-        try {
-            Class.forName("org.springframework.beans.factory.BeanFactory");
-        } catch (Exception e) {
-            springInClasspath = false;
-        }
+        Class c = Class.forName(args[0]);
 
-        Server server;
-        try {
-            if (springInClasspath) {
-                // run spring function server
-                server = new SpringFunctionServer(args);
-            } else {
-                // run plain old java function server
-                server = new POJFunctionServer(args[0], args[1]);
-            }
-        } catch (Exception ex) {
-            server = new ErrorServer(ex, springInClasspath);
```

Comment:
If removing the `ErrorServer`, you should also remove all the other `ErrorServer` related files. But this will run into the issue where if a function file is invalid, the function might still report as `"Ready"` instead of `"Error"` but this should be handled in a separate issue.

Reply:
Agreed about deleting dead code.

The intent of this change is to fail fast: if the function-server cannot be started, it should exit with an error.